ui/src/App.tsx

```diff
@@ -1,8 +1,4 @@
 import { Suspense, useCallback, useEffect, useRef, useState } from "react";
-// @ts-ignore
-import { app } from "/scripts/app.js";
-// @ts-ignore
-import { api } from "/scripts/api.js";
```

Comment:
instead of changing all app to window.app can you just
```
const app = window.app

```
and use app as before below

Reply:
already edited